lms/djangoapps/courseware/tests/test_video_handlers.py

```diff
@@ -1234,27 +1237,27 @@ def test_good_txt_transcript(self):
 
         _upload_sjson_file(good_sjson, self.item.location)
         self.item.sub = _get_subs_id(good_sjson.name)
-        transcripts = self.item.get_transcripts_info()
-        text, filename, mime_type = self.item.get_transcript(transcripts, transcript_format="txt")
+        text, filename, mime_type = get_transcript(self.item, output_format=Transcript.TXT)
         expected_text = textwrap.dedent("""\
             Hi, welcome to Edx.
             Let's start with what is on your screen right now.""")
 
         self.assertEqual(text, expected_text)
-        self.assertEqual(filename, self.item.sub + '.txt')
+        self.assertEqual(filename, 'en_' + self.item.sub + '.txt')
         self.assertEqual(mime_type, 'text/plain; charset=utf-8')
 
     def test_en_with_empty_sub(self):
 
-        transcripts = {"transcripts": {}, "sub": ""}
+        self.item.sub = ""
+        self.item.transcripts = None
         # no self.sub, self.youttube_1_0 exist, but no file in assets
         with self.assertRaises(NotFoundError):
-            self.item.get_transcript(transcripts)
+            get_transcript(self.item)
 
         # no self.sub and no self.youtube_1_0, no non-en transcritps
         self.item.youtube_id_1_0 = None
-        with self.assertRaises(ValueError):
-            self.item.get_transcript(transcripts)
+        with self.assertRaises(NotFoundError):
```

Comment:
It still raises an error, but not the same. I think this should be fine, but flagging as a potential breaking change anyway.

Reply:
This side effect looks like a positive improvement, to me.  The exception more accurately describes what happened.